### PR TITLE
pyfa.pyfa version 2.35.0

### PR DESCRIPTION
**Pull request by chiploonkar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16027)

### CODE REVIEW

File: manifests/p/pyfa/pyfa/2.35.0/pyfa.pyfa.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: pyfa.pyfa
+PackageVersion: 2.35.0
+PackageLocale: en-US
+Publisher: pyfa
+PackageName: pyfa
+License: GNU General Public License v3.0
+ShortDescription: pyfa Setup
```
Comment (ItzLevvie) [inline manifests/p/pyfa/pyfa/2.35.0/pyfa.pyfa.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Python fitting assistant, cross-platform fitting tool for EVE Online
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]
image

**Comment by praetor29** [comment]
I'm pretty new to this, so I had a question — how would I go about updating this when a new version of the application comes out? This time I used `wingetcreate new` with the installer link. Would I use `wingetcreate update` next time?

Thanks!

**Comment by denelon** [comment]
@praetor29 yes, that is what we're intending with the `wingetcreate update` command.

**Pull request merged**